OpenOversight/app/models/database.py

```diff
@@ -39,22 +39,26 @@
 class Department(BaseModel):
     __tablename__ = "departments"
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(255), index=True, unique=True, nullable=False)
+    name = db.Column(db.String(255), index=False, unique=False, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
+    state = db.Column(db.String(2), index=True, server_default="")
```

Comment:
I don't think we should add an index on state.
We should also decide on whether the default is the empty string (and then make the column non-null) or leave null as the default (combining both should be avoided). Either works for me